pkg/sql/logictest/testdata/logic_test/interval

```diff
@@ -367,3 +367,22 @@ subtest regression_62369
 
 query error "10000000000000000000000000000000000": value out of range
 SELECT INTERVAL '10000000000000000000000000000000000 year'
+
+statement ok
+CREATE TABLE regression_66118 (
```

Comment:
can you make this

```
query TT
SELECT i / 2 FROM ( VALUES
  ('0 days 0.253000 seconds'::interval),
  (INTERVAL '0.000001'::interval)
) regression_66118(i)
----

```

this expresses the test in a fewer number of lines.

Reply:
Should the create table statement be needed?